generate/generate.go

```diff
@@ -238,6 +238,16 @@ func New(os string) (generator Generator, err error) {
 			Seccomp: seccomp.DefaultProfile(&config),
 		}
 	}
+	if os == "freebsd" {
```

Comment:
```suggestion
	} else if os == "freebsd" {
```

I think the pattern of 

```
if os == "linux" {
	// ...
}
if os == "freebsd" {
	// ...
}
```

is a bit odd, since those conditionals are mutually exclusive.  An if/else if pattern or a switch might be more appropriate here.